en/mulesoft/modules/ROOT/pages/ms_composer_workday_reference.adoc

```diff
@@ -40,6 +40,12 @@ The password that you use to log in to Workday.
 Host Name::
 
 The host name of your Workday instance. For example, `wd2-impl-services1.workday.com`.
++
+To find your host name:
+
+. Log into Workday via the web interface.
+. Go to *Public Web Service*.
```

Comment:
here we can describe it because there workday uses mainly search box to perform any actions so maybe better would be:
Search for Public Web Service in the search box and select it from search results.